backend/src/handlers/signin/SigninHandler.ts

```diff
@@ -1,29 +1,40 @@
-import type { Response } from 'express';
+import express from 'express';
+import type { Request, Response } from 'express';
 
 import { Result } from '@infra/Result';
+import { SigninErrorCodes } from '@shared/signin/types/Signin';
 
-import { SigninErrorCodes } from '@shared/api/signin/types/Signin';
-import { BaseHandler } from '../BaseHandler';
+import { Logger } from '../../utils/Logger';
 import { UserService } from '../../users/UserService';
+import { type SigninPayload, type SigninOutput, signinSchema } from '../../shared/signin/types/Signin';
+import { BaseHandler } from '../BaseHandler';
 
-import { type SigninPayload, type SigninOutput, signinSchema } from '../../shared/api/signin/types/Signin';
+export const router = express.Router();
 
 /**
  * SigninHandler signs in a user
  */
 class SigninHandler extends BaseHandler<SigninPayload, SigninOutput> {
   constructor() {
-    super(signinSchema, SigninErrorCodes);
+    super(signinSchema, SigninErrorCodes, false);
```

Comment:
@tommy-trombone @j-phey passing `false` for "requires auth" in the `Sign in handler`